Move to testing with pycodestyle instead of pep8

[Guido recommended](https://github.com/PyCQA/pycodestyle/issues/466) not using pep8 to describe the module to test for PEP8 compliance. Testing using the pep8 module ought to be move to use [pycodestyle](https://github.com/PyCQA/pycodestyle) instead. Supposedly the package is feature-equivalent with pep8 but that is not longer true per [my experience](https://github.com/oduwsdl/ipwb/issues/279). Additional tests may have been added to pep8/pycodestyle to be stricter to the PEP8 style guide.

Some Travis build ouput:
```
pep8 has been renamed to pycodestyle (GitHub issue #466)
Use of the pep8 tool will be removed in a future release.
Please install and use `pycodestyle` instead.
```
So I suppose switching over is a good idea at this point. I'm also getting E722 in a few files:
```
(venv) Grants-MBP:CarbonDate gatkins$ pycodestyle --ignore=E501 main.py ./modules/
./modules/cdGetArchives.py:139:5: E722 do not use bare except'
./modules/cdGetBacklinks.py:37:5: E722 do not use bare except'
./modules/cdGetBacklinks.py:66:5: E722 do not use bare except'
./modules/cdGetBing.py:29:5: E722 do not use bare except'
./modules/cdGetBitly.py:30:5: E722 do not use bare except'
./modules/cdGetBitly.py:59:9: E722 do not use bare except'
./modules/cdGetBitly.py:129:5: E722 do not use bare except'
./modules/cdGetFirstAppearanceInArchives.py:25:5: E722 do not use bare except'
./modules/cdGetFirstAppearanceInArchives.py:110:5: E722 do not use bare except'
./modules/cdGetGoogle.py:99:9: E722 do not use bare except'
./modules/cdGetLowest.py:45:9: E722 do not use bare except'
./modules/cdGetLowest.py:83:9: E722 do not use bare except'
./modules/cdGetPubdate.py:18:5: E722 do not use bare except'
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -9,7 +9,7 @@ install:
   - pip install pytest-cov
 # command to run tests
 script:
-  - pycodestyle --ignore=E501 main.py .
+  - pycodestyle . --ignore=E501,W605,W504,W503 ./main.py
   - py.test -s --cov=./
 after_success:
   - codecov

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,12 +4,12 @@ python:
 # command to install dependencies
 install:
   - pip install -r requirements.txt
-  - pip install pep8
+  - pip install pycodestyle
   - pip install codecov
   - pip install pytest-cov
 # command to run tests
 script:
-  - pep8 --ignore=E501 main.py
+  - pycodestyle --ignore=E501 main.py .
   - py.test -s --cov=./
 after_success:
   - codecov

--- a/modules/cdGetArchives.py
+++ b/modules/cdGetArchives.py
@@ -136,7 +136,7 @@ def getArchives(url, outputArray, outputArrayIndex, verbose=False, **kwargs):
         logging.debug("Done Archives 1")
         return archives
 
-    except:
+    except Exception:
         logging.exception(sys.exc_info())
         earliest = ""
         outputArray[outputArrayIndex] = earliest

--- a/modules/cdGetBacklinks.py
+++ b/modules/cdGetBacklinks.py
@@ -34,7 +34,7 @@ def getBacklinks(url):
             url = page[loc + len(start_str):fin]
             inlinks.append(url)
             loc = fin
-    except:
+    except Exception:
         logging.exception('cdGetBacklinks :', sys.exc_info())
 
     return inlinks
@@ -63,7 +63,7 @@ def getBacklinksFirstAppearanceDates(url, outputArray, outputArrayIndex,
 
             if(epoch < lowest_epoch):
                 lowest_epoch = epoch
-    except:
+    except Exception:
         logging.exception('cdGetBacklinks :', sys.exc_info())
 
     if(lowest_epoch == 99999999999):

--- a/modules/cdGetBing.py
+++ b/modules/cdGetBing.py
@@ -27,7 +27,7 @@ def getBing(url, outputArray, indexOfOutputArray, verbose=False, **kwargs):
 
             apiKey = json.loads(config)
             apiKey = apiKey['BingAPIKey']
-    except:
+    except Exception:
         logging.debug('cdGetBing: ', sys.exc_info())
         return ''
 

--- a/modules/cdGetBitly.py
+++ b/modules/cdGetBitly.py
@@ -27,7 +27,7 @@ def GetBitlyJson(URL):
 
             ACCESS_TOKENs = json.loads(config)
             ACCESS_TOKENs = ACCESS_TOKENs['AccessToken']
-    except:
+    except Exception:
         logging.exception('cdGetBitly: %s', sys.exc_info())
         return ''
 
@@ -56,7 +56,7 @@ def GetBitlyJson(URL):
 
         try:
             jsonData = json.loads(page.decode())
-        except:
+        except Exception:
             logging.debug('Failed to load Bitly json')
             continue
 
@@ -126,7 +126,7 @@ def getBitly(url, outputArray, indexOfOutputArray, verbose=False, **kwargs):
         logging.debug("Done Bitly")
         return str(creation_date)
 
-    except:
+    except Exception:
         logging.exception('cdGetBitly: %s', sys.exc_info())
         outputArray[indexOfOutputArray] = ""
         kwargs['displayArray'][indexOfOutputArray] = ""

--- a/modules/cdGetFirstAppearanceInArchives.py
+++ b/modules/cdGetFirstAppearanceInArchives.py
@@ -22,7 +22,7 @@ def getWebpage(url):
     try:
         response = requests.get(url, headers=headers)
         return response
-    except:
+    except Exception:
         return None
 
 
@@ -107,7 +107,7 @@ def getFirstAppearance(url, inurl):
             if end == (start + 1):
                 return date
 
-    except:
+    except Exception:
 
         logging.exception(sys.exc_info())
         return ""

--- a/modules/cdGetGoogle.py
+++ b/modules/cdGetGoogle.py
@@ -96,7 +96,7 @@ def genericGetCreationDate(page):
             date = time.strftime('%Y-%m-%dT%H:%M:%S',
                                  time.gmtime(epoch))
             allDates.append(date)
-        except:
+        except Exception:
             pass
 
     return getLowest(allDates)

--- a/modules/cdGetLowest.py
+++ b/modules/cdGetLowest.py
@@ -42,7 +42,7 @@ def getLowest(dates):
 
             if(epoch < lowest_epoch):
                 lowest_epoch = epoch
-        except:
+        except Exception:
             continue
 
     if(lowest_epoch == 99999999999):
@@ -80,7 +80,7 @@ def getLowestSources(sources):
                 earliest_sources = [source]
             elif(epoch == lowest_epoch):
                 earliest_sources.append(source)
-        except:
+        except Exception:
             continue
 
     return earliest_date, earliest_sources

--- a/modules/cdGetPubdate.py
+++ b/modules/cdGetPubdate.py
@@ -15,7 +15,7 @@ def parseStrDate(dateString):
     try:
         dateTimeObj = dateutil.parser.parse(dateString)
         return dateTimeObj
-    except:
+    except Exception:
         return None
 
 

